roles/kubernetes/preinstall/tasks/main.yml

```diff
@@ -3,6 +3,11 @@
   tags:
     - asserts
 
+- import_tasks: pre_upgrade.yml
+  tags:
+    - upgrade
+
+
```

Comment:
This needs to move below the next step that sets the bin directory for CoreOS. That's why it's failing CI.